docs/projects.md

```diff
@@ -43,6 +43,18 @@
 [Strimzi](https://github.com/strimzi)|Alexis Richardson, Matt Klein|[4/9/19](https://github.com/cncf/toc/pull/266)|[8/28/19](https://github.com/strimzi)|Sandbox
 [KubeVirt](https://github.com/kubevirt)|Brian Grant, Liz Rice|[7/9/19](https://github.com/cncf/toc/pull/265)|[9/9/19](https://github.com/kubevirt)|Sandbox
 [Longhorn](https://github.com/longhorn)|Alexis Richardson, Liz Rice|[7/9/19](https://github.com/cncf/toc/pull/259)|[10/11/19](https://github.com/longhorn/longhorn)|Sandbox
+## Graduated Projects
+
+
+**Project**|**Sponsor**|**TOC Deck**|**Graduated**|**Maturity Level**
+:-----:|:-----:|:-----:|:-----:|:-----:
+[Kubernetes](https://kubernetes.io/)|Alexis Richardson|N/A|3/06/18|Graduated
+[Prometheus](https://prometheus.io/)|Alexis Richardson|[3/4/16](https://docs.google.com/presentation/d/1GtVX-ppI95LhrijprGENsrpq78-I1ttcSWLzMVk5d8M/edit?usp=sharing)|8/9/18 |Graduated
+[Fluentd](http://www.fluentd.org/)|Brian Grant|[8/3/16](https://docs.google.com/presentation/d/1S79MNv3E2aG8nuZJFJ0XMSumf7jnKozN3vdrivCH77U/edit?usp=sharing)| 4/11/19 |Graduated
+[CoreDNS](https://coredns.io/)|Jonathan Boulle|[8/17/16](https://docs.google.com/presentation/d/1LPvM44Pi7gletiDs40P7XmTKJLez5nz88ObYCHrHal8/edit?usp=sharing)| 1/24/19 |Graduated
+[containerd](https://containerd.io/)|Brian Grant|[3/15/17](https://docs.google.com/presentation/d/1qmGsmARyMhRLwbFWG7LXJSsDHm45nqZ_QtBv5SnQL54/edit?usp=sharing)| 2/28/19 |Graduated
+[Envoy](https://github.com/envoyproxy/envoy)|Alexis Richardson|[8/15/17](https://goo.gl/iSP394)|11/28/19 |Graduated
+[Vitess](https://github.com/vitessio/vitess)|Brian Grant|[4/19/17](https://goo.gl/6nmyDn)| 11/05/19|Graduated 
```

Comment:
* Jaeger graduated 10/31/2019
* please use yyyy-mm-dd format for dates, which are locale-neutral
* wouldn't it make more sense to list graduated projects at the top and not duplicate them in another table?

Reply:
Dates are consistent with the rest of the formatting, will add Jaeger.